lib/configs/react.js

```diff
@@ -42,5 +42,13 @@ module.exports = {
         fieldset: ['radiogroup', 'presentation'],
       },
     ],
+    'jsx-a11y/no-redundant-roles': [
+      'error',
+      {
+        nav: ['navigation'], // default in eslint-plugin-jsx-a11y
```

Comment:
Just curious, is this needed because the object is overridden when we provide our own config/mapping? 🤔

Reply:
Yes, that is correct!

Reply:
Or at least thats how I remember it.. let me double check 👀

Reply:
If I'm interpreting the [logic](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/src/rules/no-redundant-roles.js#L60) correctly, it looks like the rule will use the default `{ nav: ['navigation']}` _if_ the consumer isn't already passing an alternate option! This seems fine to keep!